### PR TITLE
modulesync 7.2.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 6.0.0 - 7.2.0

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -31,12 +31,6 @@
         "18.04",
         "20.04"
       ]
-    },
-    {
-      "operatingsystem": "Debian",
-      "operatingsystemrelease": [
-        "9"
```
Comment (ekohl) [inline metadata.json:38]: Usually we submit a separate PR for this and include it in the modulesync PR. I'd also like to add current Debian versions, provided they pass of course.

Comment (zilchms) [reply metadata.json:38]: ok, i can do a separate PR to remove debian9. Tests for debian 10 and 11 are unfortunately not passing.
Those i would readd in a separate PR, too. Then we have an up-to-date testsuite to cover them. Would this be acceptable?

Comment (zilchms) [reply metadata.json]: moved the debian 9 removal to a separate PR and opened another PR for adding  debian 10 and 11 that includes this modulesync PR
